### ISSUE TITLE
[Bug]: Window rules with space(s) not working?

### ISSUE THREAD

**Issue by xfzv** [Bismuth-Forge/bismuth#362]
### Summary

Rules for windows name that don't contain any space work fine, e.g. `ark,dolphin,kcalc`. However, I cannot get rules with space(s) working, such as:
- With classes: `VirtualBox Machine,VirtualBox Manager`
- With titles: `About Mozilla Firefox`

### Steps to Reproduce

1. System Settings > Window Management > Window Tiling > Window Rules
2. Float Windows > With classes: `VirtualBox Machine,VirtualBox Manager`
3. Float Windows > With titles: `About Mozilla Firefox`
4. Open a new instance of VirtualBox Manager or open "Help > About Firefox"
5. Both windows aren't floating

### Expected behavior

I'd like to be able to add rules for windows classes/titles which contain space(s).

### Bismuth version

3.1.1-1

### KDE Plasma version

5.24.5

### The platform KWin is running on

X11

### Additional context

It's supposedly fixed with https://github.com/Bismuth-Forge/bismuth/pull/342 so maybe I'm dumb?

**Comment by benemorius** [comment]
Can you try using lowercase?

I hit this bug too and looked in to it. I get capitalized class names from `xprop WM_CLASS` but in bismuth console output they're lowercase. I changed `VirtualBox Machine` to `virtualbox machine` and it works for me.

Incidentally the spaces are handled perfectly now. All of these are valid bismuth configurations:

```
systemsettings,virtualbox machine
systemsettings, virtualbox machine
systemsettings, virtualbox machine,
```

**Comment by xfzv** [comment]
> Can you try using lowercase?
> 
> I hit this bug too and looked in to it. I get capitalized class names from `xprop WM_CLASS` but in bismuth console output they're lowercase. I changed `VirtualBox Machine` to `virtualbox machine` and it works for me.
> 
> Incidentally the spaces are handled perfectly now. All of these are valid bismuth configurations:
> 
> ```
> systemsettings,virtualbox machine
> systemsettings, virtualbox machine
> systemsettings, virtualbox machine,
> ```

Indeed, that works. However, `about mozilla firefox` works the first time I open the about Firefox menu but if I do it again, the window is tiled, until I close Firefox.

Another workaround is to use `Virtual,About` for `With titles` rules (same behavior but I prefer the more precise rule with lower cases you suggested).
